ReText/__main__.py

```diff
@@ -92,6 +93,18 @@ def main():
 	fileNames = list(map(canonicalize, sys.argv[1:]))
 	previewMode = False
 	readStdIn = False
+    
+	if not QDBusConnection.sessionBus().registerService('org.ReText'):
+		print('Send file to main instance')
+		iface = QDBusInterface('org.ReText', '/', '', QDBusConnection.sessionBus())
+		msg = iface.call('openFileWrapper', sys.argv[1] if len(sys.argv) > 1 else "")
+		QDBusReply(msg)
+		sys.exit(1)
+
+	connection = QDBusConnection.sessionBus()
+	connection.registerObject('/', window, QDBusConnection.ExportAllSlots)
+	connection.registerService('org.ReText')
```

Comment:
The connection is already registered on line 97, isn’t it? Can we leave only one of these lines?

And maybe also define `connection` earlier and don’t call `sessionBus()` twice.